integration/test/basic/basic_test.go

```diff
@@ -14,35 +14,53 @@ import (
 	"github.com/giantswarm/chart-operator/integration/templates"
 )
 
-func TestChartInstalled(t *testing.T) {
-	err := f.InstallResource("chart-operator-resource", templates.ChartOperatorResourceValues, ":stable")
+func TestChartLifecycle(t *testing.T) {
+	const testRelease = "tb-release"
+	const cr = "chart-operator-resource"
+
+	log.Printf("creating %q", cr)
+	err := f.InstallResource(cr, templates.ChartOperatorResourceValues, ":stable")
+	if err != nil {
+		t.Fatalf("could not install %q %v", cr, err)
+	}
+
+	err = waitForReleaseStatus(helmClient, testRelease, "DEPLOYED")
+	if err != nil {
+		t.Fatalf("could not get release status of %q %v", testRelease, err)
+	}
+	log.Printf("%q succesfully deployed", testRelease)
+
+	log.Printf("deleting %q", cr)
+	err = helmClient.DeleteRelease(cr)
 	if err != nil {
-		t.Fatalf("could not install chart-operator-resource-chart %v", err)
+		t.Fatalf("could not delete %q %v", cr, err)
 	}
 
-	var rc *helmclient.ReleaseContent
+	err = waitForReleaseStatus(helmClient, testRelease, "DELETED")
+	if !helmclient.IsReleaseNotFound(err) {
```

Comment:
Release gets purged so in fact we are don't expect a "DELETED" Status, but expecting a error. Deletion takes a bit so we use the `waitForReleaseStatus` to wait for the error